generators/docker-compose/index.js

```diff
@@ -154,6 +154,10 @@ module.exports = yeoman.Base.extend({
                     var path = this.destinationPath(this.directoryPath + this.appsFolders[i]+'/.yo-rc.json');
                     var fileData = this.fs.readJSON(path);
                     var config = fileData['generator-jhipster'];
+                    //this currently can happen, because the .yo-rc doesn't have a baseName
+                    if(fileData['generator-jhipster'].baseName === undefined) {
```

Comment:
All application type should set the` baseName` and hence this should never happen, for appType `uaa` we should set the basename while generating the app appropriately in server or app generator